### DISCUSSION TITLE
How to scale up the dify deployment

### DISCUSSION THREAD

**Discussion by 1559550282** [langgenius/dify#5038] [General]
Hi, Folks,
If I deploy Dify with Docker compose, and encountered workload increasing, how can I scale up the deployment? Can I add more dockers with the same API entry? Or can I deploy Dify in K8S?

**Comment by patryk20120** [comment]
Depends how you want to scale it (horizontally or vertically). The best solution would be deploying onto k8s (here is a helm chart available https://github.com/langgenius/dify/discussions/3769 ) and scale overloaded containers e.g. `api` when desired metrics (custom or resources) will increase.
